traffic-policy/actions/oauth/config.mdx

```diff
@@ -0,0 +1,141 @@
+| -------------------------------------- | --------------------------------------------------------------------------- |
+| `ngrok-auth-identity-provider-user-id` | Provider-defined identifier of the authorized user.                         |
+| `ngrok-auth-identity-name`             | Full name of the authorized user.                                           |
+| `ngrok-auth-identity-email`            | Authorized user's primary email address.                                    |
+| `ngrok-auth-access-token`              | The user's OAuth access token. Undefined when using a managed application.  |
+| `ngrok-auth-refresh-token`             | The user's OAuth refresh token. Undefined when using a managed application. |
+
+### Special Paths
+
+Upstream applications behind endpoints with this module enabled do not receive
+any requests to paths beginning with `/auth/`. Your application may redirect
+clients to the following paths to invoke different behaviors.
+
+| Path           | Description                                                                                                                                                                                                                                                                                                                     |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `/auth/login`  | Redirect users to this path to explicitly begin an authentication flow. After authentication, users will be redirected to `/`. If the IdP supports it, ngrok will attempt to instruct the IdP to force re-authentication which will force users to re-enter their credentials with the IdP even if they were already logged in. |
+| `/auth/logout` | Logs the user out by clearing their session cookie. Redirect users to this path to log them out.                                                                                                                                                                                                                                |
+
+### Events
+
+When this module is enabled, it populates the following fields in the
+[http_request_complete.v0](/obs/reference/#http-request-complete) event:
+
+| Fields                |
+| --------------------- |
+| `oauth.app_client_id` |
+| `oauth.decision`      |
+| `oauth.user.id`       |
+| `oauth.user.name`     |
+
+### Errors
+
+:::note Coming Soon
+
+This documentation is incomplete. Please check back later, we appreciate your
+patience.
+
+:::
+
+### Edges
+
+OAuth is an HTTPS Edge module which can be applied to Routes.
+
+- [OAuth Edge Module API Resource](/api/resources/edge-route-o-auth-module/)
+
+### Pricing
+
```

Comment:
@Abdiramen and @samcrichard this section is empty and I believe it should talk about MaUs?